pkgs/top-level/all-packages.nix

```diff
@@ -4007,6 +4007,8 @@ with pkgs;
 
   ios-webkit-debug-proxy = callPackage ../development/mobile/ios-webkit-debug-proxy { };
 
+  weblate = callPackage ../servers/web-apps/weblate { };
```

Comment:
I guess `pkgs/servers/web-apps/weblate/default.nix` should be moved to `pkgs/by-name/we/weblate/package.nix` and this line should be removed.

Reply:
Makes sense, pushed that.